xmind/tests/test_sheet_element.py

```diff
@@ -0,0 +1,402 @@
+import logging
+from xmind.core.sheet import SheetElement
+from xmind.tests import base
+from unittest.mock import patch, Mock, MagicMock
+from xmind.core.const import *
+
+
+
+class TestSheetElement(base.Base):
+    """Test class for SheetElement class"""
+
+    def getLogger(self):
+        if not getattr(self, '_logger', None):
+            self._logger = logging.getLogger('sheetElement')
+        return self._logger
+
+    def setUp(self):
+        super(TestSheetElement, self).setUp()
+        self._init_method = self._init_patch_with_name('_init', 'xmind.core.sheet.WorkbookMixinElement.__init__')
```

Comment:
you have to check if your methods were called in every tests.
for this purpose I recommend you to create additional method to check if methods were called.

Reply:
Why? I check this method in test_init. If this method is failed I will notice this in test_init

Reply:
what the reason to mock method and not check it then?